pom.xml

```diff
@@ -1006,6 +1010,10 @@
               <goal>integration-test</goal>
               <goal>verify</goal>
             </goals>
+            <configuration>
+              <excludeGroups>${accumulo.integrationTestsExcludedGroups}</excludeGroups>
+              <groups>${accumulo.integrationTestsGroups}</groups>
+            </configuration>
```

Comment:
This is a great way to add indirection to the properties, so users can control failsafe directly, without affecting surefire. Unfortunately, surefire and failsafe use the same user properties for these, so we must add this indirection ourselves (I lament that surefire doesn't use `tests.groups` and `tests.excludeGroups` and failsafe doesn't use `it.groups` and `it.excludeGroups`).

Since you've added this indirection for failsafe, would it make sense to also add this to surefire? Right now, the way to control surefire is with the `groups` and `excludeGroups` properties respected by the plugin. That will override these settings for failsafe, too, and that's probably not desirable.